src/package-linker.js

```diff
@@ -446,6 +446,24 @@ export default class PackageLinker {
       }
       const ref = pkg._reference;
       invariant(ref, 'Package reference is missing');
+      // TODO: We are taking the "shortest" ref tree but there may be multiple ref trees with the same length
```

Comment:
Is it a problem if we choose one of those at random?

Reply:
I honestly don't know, hence the TODO.